index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>youtube</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="youtube.css">
+        <link rel="icon"  href="youtube.png">
+    </head>
+    <body>
+     
+        <div id="nav-bar">
```

Comment:
please  change div tag to <nav>,it's better to use semantic HTML tags

Reply:
Please ensure that all your comments are clear and easy to understand. 
For Example:
This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?. If you wish to use  HTML tags like this `<p>` in your comments, try to wrap it in back-ticks.

Kindly make a necessary correction to the comment to state clearly the changes requested. What do I change the `div` tag to?